lib/src/screen/dice/dice_screen.dart

```diff
@@ -1,17 +1,31 @@
+import 'dart:async';
+import 'package:first_flutter_app/src/styles/dimensions.dart';
```

Comment:
use relative imports for your own classes/files

Reply:
Done!